desc/objectives/linear_objectives.py

```diff
@@ -673,6 +673,11 @@ def build(self, eq, use_jit=True, verbose=1):
             Level of output.
 
         """
+        if eq.pressure is None:
```

Comment:
maybe a stupid question, but if a user has supplied kinetic profiles to an equilibrium but then just wants the pressure to be fixed without caring about the kinetic profiles... this probably is a bad use case is that ever a case we should worry about supporting? would mean nonlinear constraint tho since it is a constraint on the products of T and n

Reply:
Yeah in that case it gets a lot trickier. I don't think it would be a common use case that we need to accommodate, at least not for now.